ui/pages/settings/flask/snaps-list-tab/index.scss

```diff
@@ -0,0 +1,13 @@
+.snaps-list-wrapper {
+  width: auto;
+}
+
+.snap-settings-card {
+  margin-top: 8px;
+  margin-bottom: 8px;
```

Comment:
If there's no margin on the left or right, we can shorthand with `margin: 8px 0`, but no big deal.